boards/xtensa/yd_esp32/yd_esp32.dts

```diff
@@ -0,0 +1,198 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+/dts-v1/;
+
+#include <espressif/esp32.dtsi>
```

Comment:
this needs to be altered as the path is no longer available on current upstream

Reply:
@marekmatej , @sylvioalves,

The YD-ESP32 board supports [^1] the following modules:

- ESP32-WROOM-32E (N4/N8/N16)
- ESP32-WROOM-32UE (N4/N8/N16)

On the [ESP32-DEVKITC-WROOM](https://docs.zephyrproject.org/latest/boards/xtensa/esp32_devkitc_wroom/doc/index.html) board which supports the same modules as the YD-ESP32 board, it is being used  `#include <espressif/esp32/esp32_wroom_32ue_n4.dtsi>`:

https://github.com/zephyrproject-rtos/zephyr/blob/438bd1db3d9d779cc1cbf29cd97d5b7bce3b3e43/boards/xtensa/esp32_devkitc_wroom/esp32_devkitc_wroom.dts#L8

Questions:
1. Would I have to change the line `#include <espressif/esp32.dtsi>`  to one of the following options? Or is there another option when the devkit will be able to support different sizes of flash memory?
    - `#include <espressif/esp32/esp32_wroom_32ue_n4.dtsi>` ([esp32_wroom_32ue_n4.dtsi](https://github.com/zephyrproject-rtos/zephyr/blob/main/dts/xtensa/espressif/esp32/esp32_wroom_32ue_n8.dtsi))
    - `#include <espressif/esp32/esp32_wroom_32ue_n8.dtsi>` ([esp32_wroom_32ue_n8.dtsi](https://github.com/zephyrproject-rtos/zephyr/blob/main/dts/xtensa/espressif/esp32/esp32_wroom_32ue_n8.dtsi))
    - `#include <espressif/esp32/esp32_wroom_32ue_n16.dtsi>` ([esp32_wroom_32ue_n16.dtsi](https://github.com/zephyrproject-rtos/zephyr/blob/main/dts/xtensa/espressif/esp32/esp32_wroom_32ue_n8.dtsi))

2. Is there any relevance, since there is is a Flash size detection tool?

![Screenshot02_Zephyr_2023-08-05](https://github.com/zephyrproject-rtos/zephyr/assets/103332557/b1f28bd1-3e10-4a27-b41c-384dd343b321)

---

[^1]: As stated in the [product page](https://pt.aliexpress.com/item/1005004363550813.html);

Reply:
@jcsxdev Yes, boards based on ESP32 SIP, can have different configurations. The auto-detecting feature for Flash (and PSram) gives you runtime information, but since Zephyr drivers are configured from the dts during the build time we are providing this statically.
For your board, I'd select the biggest available SIP, or the one that would be assembled most of the time.

Reply:
> For your board, I'd select the biggest available SIP, or the one that would be assembled most of the time.

Done.

https://github.com/zephyrproject-rtos/zephyr/blob/28912acc4cc733afcc2f75994251b70760949f5c/boards/xtensa/yd_esp32/yd_esp32.dts#L1-L6